OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs

```diff
@@ -95,7 +95,7 @@ public async Task ShouldThrowServiceExceptionOnAddWhenExceptionOccursAndLogItAsy
             var serviceException = new Exception();
 
             var failedCalendarEntryAttachment =
-                new FailedCalendarEntryAttachment(serviceException);
+                new FailedCalendarEntryServiceAttachment(serviceException);
```

Comment:
Sorry for the typo previously, it is Failed<Entity>ServiceException so it is FailedCalendarEntryAttachmentServiceException